common-items.lic

```diff
@@ -119,9 +119,44 @@ module DRCI
     result =~ /inside/
   end
 
+  # Is the item in either hand?
   def in_hands?(item)
-    item_regex = item.split.size > 1 ? /.*\b#{item.split.first}.*\b#{item.split.last}/i : /\b#{item}/i
-    item_regex =~ "#{DRC.right_hand}#{DRC.left_hand}"
+    in_hand?(item, 'either')
+  end
+
+  # Is the item in your left hand?
+  def in_left_hand?(item)
+    in_hand?(item, 'left')
+  end
+
+  # Is the item in your right hand?
+  def in_right_hand?(item)
+    in_hand?(item, 'right')
+  end
+
+  # Checks if the item is in one or more hands.
+  # Hand options are: left, right, either, both.
+  def in_hand?(item, whichHand = 'either')
+    item_regex = item.split.size > 1 ? /#{item.split.first}.*\b#{item.split.last}/i : /\b#{item}/i
+    item_found = false
+    case whichHand.downcase
+    when 'left'
+      # Your holding the item in your left hand?
+      item_found = ("#{DRC.left_hand}" =~ item_regex) != nil
```

Comment:
I don't feel strongly in favor of ruby lang convention of predicate methods returning true/false instead of truthy/falsey values, but if you want to conform to it the double bang approach is clean and idiomatic. Also I think DRC.left_hand is already returning a string so the interpolation `"#{}` is no longer needed since you aren't combining two strings together anymore.

`item_found = ("#{DRC.left_hand}" =~ item_regex) != nil`
combined with the other suggestion becomes
`!!(DRC.left_hand =~ item_regex)`

Which gets back to my minor preference for no strict true/false adherence because then you just get away with
`DRC.left_hand =~ item_regex`